### PR TITLE
Inline and modify connect with us section for landing pages

### PR DESCRIPTION
**Pull request by kelsonic**
# Description

**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/33176

This PR switches the dependency on `fieldAdministration` and `fieldClpConnectWithUs` in favor of `fieldRelatedOffice` for the connect with us sections on landing pages and campaign landing pages.

### http://localhost:3002/health-care/

image

### http://localhost:3002/initiatives/flu/

image

## Acceptance Criteria

- [x] Switch  the dependency on `fieldAdministration` and `fieldClpConnectWithUs` in favor of `fieldRelatedOffice` for the connect with us sections on landing pages and campaign landing pages.

### CODE REVIEW

File: src/site/layouts/landing_page.drupal.liquid [hunk 1]
```diff
@@ -47,105 +48,161 @@
...
-        </va-accordion>
+                </h4>
+              {% endif %}
+
+              <section>
+                <h4>Get updates</h4>
+                  <ul class="va-nav-linkslist-list social">
+                    <li>
+                      <a href="{{ fieldRelatedOffice.entity.fieldEmailUpdatesLink.url.path }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });">
+                        <i class="fas fa-envelope vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/landing_page.drupal.liquid:156]:
```suggestion
                        <i class="fas fa-envelope vads-u-padding-right--1" aria-hidden="true"></i>
```

File: src/site/layouts/landing_page.drupal.liquid [hunk 2]
```diff
@@ -47,105 +48,161 @@
...
 
-        {% if fieldAdministration != empty and fieldAdministration.entity.name != "Records benefits hub" %}
-          {% include "src/site/components/administration-hub-rail.drupal.liquid" administration = fieldAdministration.entity %}
+                  {% for socialPlatform in socialPlatforms %}
+                    {% assign socialLink = socialLinks | get: socialPlatform  %}
+                    {% if socialLink.value %}
+                      {% if socialPlatform = "youtube_channel" %}
+                        <li>
+                          <a href="http://youtube.com/channel/{{ socialLink.value }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });">
+                            <i class="fab fa-youtube vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/landing_page.drupal.liquid:175]:
```suggestion
                            <i class="fab fa-youtube vads-u-padding-right--1" aria-hidden="true"></i>
```

File: src/site/layouts/landing_page.drupal.liquid [hunk 3]
```diff
@@ -47,105 +48,161 @@
...
+                            {{ fieldRelatedOffice.entity.fieldExternalLink.title }} {{ socialPlatform | remove: '_' | capitalize }}
+                          </a>
+                        </li>
+                      {% elsif socialPlatform == "youtube" %}
+                        <li>
+                          <a href="http://{{ socialPlatform }}.com/{{ socialLink.value }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });">
+                            <i class="fab fa-{{ socialPlatform }} vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/landing_page.drupal.liquid:182]:
```suggestion
                            <i class="fab fa-{{ socialPlatform }} vads-u-padding-right--1" aria-hidden="true"></i>
```

File: src/site/layouts/landing_page.drupal.liquid [hunk 4]
```diff
@@ -47,105 +48,161 @@
...
+                            {{ fieldRelatedOffice.entity.fieldExternalLink.title }} YouTube
+                          </a>
+                        </li>
+                      {% else %}
+                        <li>
+                          <a href="http://{{ socialPlatform }}.com/{{ socialLink.value }}" onclick="recordEvent({ 'event': 'nav-hub-rail', 'nav-path': 'Connect with us' });">
+                            <i class="fab fa-{{ socialPlatform }} vads-u-padding-right--1"></i>
```
Comment (timwright12) [inline src/site/layouts/landing_page.drupal.liquid:189]:
```suggestion
                            <i class="fab fa-{{ socialPlatform }} vads-u-padding-right--1" aria-hidden="true"></i>
```

**Comment by timwright12** [comment]
@kelsonic fixed up some icons

**Review by zacharymorel — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

**Pull request merged**